### ISSUE TITLE
Use babel for UI files

### ISSUE THREAD

**Issue by HarshKhandeparkar** [publiclab/image-sequencer#670]
### Please describe the problem (or idea)

We can use babel while compiling ui files. As we have recently started adding ui files to `dist/` directory, we can write es6 code for the ui and compile it using babel. This will allow us to shorten the long lines of step.ui which contain `\` every line break and `+` operator for every dynamic field which is loaded from module files using template literals. We can also use many other features like destructuring and arrow functiona to write efficient code.

****

**Comment by HarshKhandeparkar** [comment]
@publiclab/is-reviewers is this issue helpful?

**Comment by Divy123** [comment]
I think the idea is awesome.
Your opinion please @jywarren .

**Comment by HarshKhandeparkar** [comment]
@tech4GT @jywarren this issue can potentially shorten a lot of code but implementing this may be a bit difficult as the index.html will have to include the compiled code and not es6 code. Potentially webpack might also be required for pulling this off.

**Comment by Divy123** [comment]
Thanks @HarshKhandeparkar for opening this issue.
Can make a big change for the good.

**Comment by HarshKhandeparkar** [comment]
I have an idea, the examples/` directory can be divided into two parts the src folder and the index.html file. All the js source can go inside the src folder inside the `examples/` directory and images etc. can remain where they are.

Instead of requiring each js file from `src/` or `lib/` folder, the `index.html` can require `dist/image-sequencer-ui.min.js`. Everything else will remain the same including the `/examples/` URL if this is implemented this way.

The only problem will be that UI files will have to be watched and compiled on change too.

**Comment by jywarren** [comment]
Agreed! Actually the UI files are already compiled now, via a PR
from @Mridul97 -- and they'll be browserified too in a new PR! This is a
great idea and we could include Babel in the Gruntfile, right?

**Comment by Divy123** [comment]
Yes this is great to implement and I would like to work on this.

**Comment by jywarren** [comment]
That sounds great! Thank you.

**Comment by akshaygupta533** [comment]
I am willing to contribute to this. @HarshKhandeparkar  @jywarren  can you guys guide me through this?

**Comment by Divy123** [comment]
@akshaygupta533  I am working on this.
You may look into some other issue.
